kvrocks.conf

```diff
@@ -575,15 +575,8 @@ rocksdb.target_file_size_base 128
 # allowed.
 rocksdb.max_write_buffer_number 4
 
-# Maximum number of concurrent background compaction jobs, submitted to
-# the default LOW priority thread pool.
-rocksdb.max_background_compactions 4
-
-# Maximum number of concurrent background memtable flush jobs, submitted by
-# default to the HIGH priority thread pool. If the HIGH priority thread pool
-# is configured to have zero threads, flush jobs will share the LOW priority
-# thread pool with compaction jobs.
-rocksdb.max_background_flushes 4
```

Comment:
AFAIK it is a BREAKING CHANGE, you remove some configuration options in kvrocks.conf. You may need to explain more about this.

Reply:
Yes, I think we can mark them as deprecated first and set the default value to -1, then encourage users to use the max_background_jobs like RocksDB: https://github.com/facebook/rocksdb/blob/8c445407b79ab02620e15cdd1257b1c99ea09601/include/rocksdb/options.h#L701